### PR TITLE
	Linking github data with react client. #17, #12

### PR DESCRIPTION
**Pull request by mjzone**

### CODE REVIEW

File: web/src/services/githubservice.js [hunk 1]
```diff
@@ -1,40 +1,19 @@
 import axios from 'axios';
-var USER_TOKEN = 'YOUR TOKEN HERE';
+var USER_TOKEN = "b5d99930784b85c51e44d87263199abbcedfa035";
```
Comment (rehrumesh) [inline web/src/services/githubservice.js]: Remove the token?

Comment (mjzone) [inline web/src/services/githubservice.js]: yes. will send another PR
